docs/src/tutorials/tutorial_4.md

```diff
@@ -0,0 +1,329 @@
+    circle(Point(0, -100), 20, :fill)
+end
+```
+
+It draws one brown hair blob given the angle. We basically rotate the whole canvas and then draw the circle always at the same local position. 
+
+Now how do we draw the hair without creating an action for each blob?
+
+Well we actually create an Action for each blob but we can use a for loop for this.
+
+We can use splatting for that :wink:
+
+```julia
+[
+    Action(26:150, (args...)->hair_blob(hair_angle[i]); subactions=[
+        SubAction(1:25, appear(:fade)),
+    ]) for i=1:20
+]...,
+```
+
+We first create a vector using the `[foobar for i=1:20]` notation but as the javis function expects `foobar, foobar, ..., foobar` without the vector we use splatting like `[foobar for i=1:20]...`
+
+I think you get the idea of how to use `appear` now. Let's add some eyes and a nose quickly before we draw our first gif.
+
+```julia
+Action(30:150, (args...)->eyes(eye_centers, 10, "darkblue"); subactions=[
+    SubAction(1:15, appear(:fade)),
+]),
+Action(45:150, (args...)->poly(nose, :fill); subactions=[
+    SubAction(1:15, appear(:fade)),
+]),
+```
+
+with:
+
+```julia
+eye_centers = [Point(-40,-30), Point(40,-30)]
+nose = [O, Point(-10,20), Point(10, 20), O]
+```
+
+and
+
+```julia
+function eyes(centers, radius, color)
+    sethue(color)
+    circle.(centers, radius, :fill)
+    setcolor("white")
+    circle.(centers, radius/5, :fill)
+end
+```
+
+![Up to the nose](./assets/jarvis_nose.gif)
+
+## Using Transformations
+
+Let's give him some moving lips so he can communicate with us:
+
+```julia
+upper_lip = [Point(-40, 45), Point(40, 45)]
+lower_lip = [Point(-40, 55), Point(40, 55)]
+```
+
+These are just the outer points of the lips:
+
+```julia
+function lip(p1, p2)
+    setline(2)
+    move(p1)
+    c1 = p1 + Point(10, 10)
+    c2 = p2 + Point(-10, 10)
+    curve(c1, c2, p2)
+    do_action(:stroke)
+end
+```
+
+This function uses some more functions of the awesome Luxor package.
+
+The lips should be a little thicker than the other lines that we have drawn so far so let's set `setline(2)`. (default is 1).
+First we move to the starting point of the lip and create two control points a bit below and to the vertical center.
+
+The `curve` function is used to draw a cubic [Bézier curve](https://en.wikipedia.org/wiki/B%C3%A9zier_curve). 
+It unfortunately doesn't support the `:stroke` at the end so we have to do this with `do_action(:stroke)` this time.
+
+Now our two actions:
+
+```julia
+Action(60:150, (args...)->lip(upper_lip...); subactions=[
+    SubAction(1:15, appear(:fade)),
+    [SubAction(20i:20i+10, Translation(0, -5)) for i in 1:5]...,
+    [SubAction(20i+10:20i+20, Translation(0, 5)) for i in 1:5]...
+]),
+Action(60:150, (args...)->lip(lower_lip...); subactions=[
+    SubAction(1:15, appear(:fade)),
+    [SubAction(20i:20i+10, Translation(0, 5)) for i in 1:5]...,
+    [SubAction(20i+10:20i+20, Translation(0, -5)) for i in 1:5]...
+]),
+```
+
+Yeah I like those `...` splatting :smile:
```

Comment:
Can you replace with the correct emoji here?

Reply:
Good catch